src/plugins/intel_gpu/src/graph/network.cpp

```diff
@@ -1222,6 +1222,15 @@ void network::execute_impl(const std::vector<event::ptr>& events) {
         }
 
         execute_primitive(inst, events);
+        // {
+        //     get_stream().finish();
+        //     const std::string layer_name = inst->id();
+        //     for (size_t i = 0; i < get_primitive(inst->id())->outputs_memory_count(); i++) {
+        //         check_mem(get_primitive(inst->id())->output_memory_ptr(i),
```

Comment:
Dead code

Reply:
Yes, you're right. it is the code for debug. I removed it. thanks.